source/appModules/dllhost.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: UTF-8 -*-
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2018 NV Access Limited, Łukasz Golonka
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
+# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
+
+""" Under older builds of Windows 10 (from RTM release to  Creators Update) dllhost is used to display a properties window. 
+Read-Only edit boxes in it can contain dates that include unwanted left-to-right and right-to-left indicator characters.
+This simply imports a proper class from explorer app module, and maps it to a edit control.
+"""
+
+import appModuleHandler
+from .explorer import ReadOnlyEditBox
+
+class AppModule(appModuleHandler.AppModule):
+
+	def chooseNVDAObjectOverlayClasses(self, obj, clsList):
+		windowClass = obj.windowClassName
+
+		if windowClass == "Edit":

```

Comment:
But can we be sure that this is read-only? Should you not check the state too?

Reply:
Yes we can, because in explorer.py i am removing  RTL and LTR chars only when edit has readonly in states. I've done it this way to avoid additional check in both explorer.py and dllhost.py. The same goes for your last commend.

Reply:
My bad for not seeing this. @leonardder what do you think?